tests/test_build.py

```diff
@@ -23,3 +23,33 @@ def test_run_pymodules_install_optional_project_dir(self):
             assert run_pymodules_install(ctx, modules, project_dir) is None
         assert m_info.call_args_list[-1] == mock.call(
             'No Python modules and no setup.py to process, skipping')
+
+    def test_strip_if_debuggable(self):
+        ctx = mock.Mock()
+        ctx.python_recipe.major_minor_version_string = "python3.6"
+        ctx.get_site_packages_dir.return_value = "test-doesntexist"
+        ctx.build_dir = "nonexistant_directory"
+        ctx.archs = ["arm64"]
+        ctx.build_as_debuggable = False
+
+        modules = ["mymodule"]
+        project_dir = None
+        with mock.patch('pythonforandroid.build.info'), \
+                mock.patch('sh.Command'),\
+                mock.patch('pythonforandroid.build.open'),\
+                mock.patch('pythonforandroid.build.shprint'),\
+                mock.patch('pythonforandroid.build.current_directory'),\
+                mock.patch('pythonforandroid.build.CythonRecipe') as m_CythonRecipe, \
```

Comment:
Minor: I usually try to patch the actual method to have the least impact, hence something like:
```python
mock.patch('pythonforandroid.build.CythonRecipe.strip_object_files') as m_strip_object_files
```

Reply:
it felt safer that way, to reduce any side effects I'm not interested in and I don't really need any functionality of the recipe anyway in this test, so it was trivial to mock a bit more far-reaching